config/dev.exs

```diff
@@ -2,10 +2,6 @@ import Config
 
 # Configure your database
 config :cklist, Cklist.Repo,
-  username: "cklist",
-  password: "cklist",
-  hostname: "localhost",
-  database: "cklist_dev",
```

Comment:
There's a similar configuration in `config.test.exs`. That would also need to change, right?

Reply:
oh, indeed. let me grab that  and update this pr